src/services/RoleService.js

```diff
@@ -0,0 +1,28 @@
+import { api, getDefaultHeaders, manageError } from 'boot/axios';
+
+/**
+ * Get all roles.
+ * @returns {Promise<object[]>} Promise with an array of roles on success
+ * otherwise an error.
+ */
+export async function find() {
+  const queryParameters = 'limit=10';
+
+  return api.get(`/api/classes/_Role?${queryParameters}`, { headers: getDefaultHeaders() })
+    .catch(manageError);
+}
+
+/**
+ * Get all roles of a user.
+ * @param {string} userId - User id.
+ * @param {string} sessionToken - The current user's session token.
+ * @returns {Promise<object[]>} Return an array of roles. Role is an object from ParseServer.
+ * See documentation for more information.
+ * @see https://docs.parseplatform.org/
+ */
+export async function findByUserId(userId, sessionToken) {
```

Comment:
same comment for line 8
weird to use the verb `find`, why not the usual `get`?

Reply:
see with you, updated.